.github/workflows/main.yaml

```diff
@@ -0,0 +1,53 @@
+name: CI
+
+on:
+  push:
+    branches:
+    - main
+  pull_request:
+
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+
+jobs:
+  lint-test:
+    uses: ./.github/workflows/lint-test.yaml
+
+
+  generate-sha-tag:
+    runs-on: ubuntu-latest
+    outputs:
+      sha-tag: ${{ steps.sha-tag.outputs.sha-tag }}
+    steps:
+      - name: Create SHA Container tag
+        id: sha-tag
+        run: |
+          tag=$(cut -c 1-7 <<< $GITHUB_SHA)
+          echo "sha-tag=$tag" >> $GITHUB_OUTPUT
+
+  publish-static-preview:
+    uses: ./.github/workflows/static-preview.yaml
+    with:
+      sha-tag: ${{ needs.generate-sha-tag.outputs.sha-tag }}
```

Comment:
You will need to mark this job as needing the `generate-sha-tag` job to access `needs` here

Reply:
d4e2325d822fc21d0fc7548d01596364792597d9